Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionMicrosoftSysmonWindowsEvent.yaml

```diff
@@ -0,0 +1,139 @@
+Parser:
+  Title: Network Session Event ASIM parser for Sysmon (Event 3)
+  Version: '0.2.1'
+  LastUpdated: June 3, 2024
+Product:
+  Name: Windows Sysmon
+Normalization:
+  Schema: NetworkSession
+  Version: '0.2.5'
```

Comment:
ASIM schema "NetworkSession" version "0.2.5" is incorrect. The correct version for ASIM schema "NetworkSession" is "0.2.6"

Reply:
done